interface/patient_file/summary/stats_full.php

```diff
@@ -202,220 +181,252 @@ function newEncounter() {
+                                $statusCompute = generate_display_field(array('data_type' => '1','list_id' => 'outcome'), $row['outcome']);
+                            } elseif ($row['enddate'] == null) {
+                                $statusCompute = xlt("Active");
+                            } else {
+                                // MU3 criteria, show medical problem's with end dates as a status of Completed.
+                                $statusCompute = ($t == 'medical_problem') ? xlt("Completed") : xlt("Inactive");
+                                $resolved = ($t == "medical_problems") ? true : false;
+                            }
 
-                                // calculate the status
-                                if ($row['outcome'] == "1" && $row['enddate'] != null) {
-                                    // Resolved
-                                    $statusCompute = generate_display_field(array('data_type' => '1','list_id' => 'outcome'), $row['outcome']);
-                                } elseif ($row['enddate'] == null) {
-                                    $statusCompute = xlt("Active");
-                                } else {
-                                    if ($focustype == 'medical_problem') {
-                                        // MU3 criteria, show medical problem's with end dates as a status of Completed.
-                                        $statusCompute = xlt("Completed");
-                                    } else {
-                                        $statusCompute = xlt("Inactive");
-                                    }
-                                }
+                            $click_class = 'statrow';
+                            if ($row['erx_source'] == 1 && $focustype == 'allergy') {
+                                $click_class = '';
+                            } elseif ($row['erx_uploaded'] == 1 && $focustype == 'medication') {
+                                $click_class = '';
+                            }
 
-                                $click_class = 'statrow';
-                                if ($row['erx_source'] == 1 && $focustype == 'allergy') {
-                                    $click_class = '';
-                                } elseif ($row['erx_uploaded'] == 1 && $focustype == 'medication') {
-                                    $click_class = '';
-                                }
+                            $shortBegDate = text(trim(oeFormatShortDate($row['begdate'])));
+                            $shortEndDate = text(trim(oeFormatShortDate($row['enddate'])));
+                            $fullBegDate = text(trim(oeFormatDateTime($row['begdate'])));
+                            $fullEndDate = text(trim(oeFormatDateTime($row['enddate'])));
+                            $shortModDate = text(trim(oeFormatShortDate($row['modifydate'])));
+                            $fullModDate = text(trim(oeFormatDateTime($row['modifydate'])));
```

Comment:
above 6 lines are a good example why should instead do the html escaping (ie. do not do the text() above) when actually echo them (since in some places should be using attr() instead of text() below)